pages/index.jsx

```diff
@@ -0,0 +1,26 @@
+import Button from "../components/button/button";
+import Link from "next/link";
+import MainContainer from "../container/mainContainer/mainContainer";
+import IndexCard from "../components/indexCard/IndexCard";
+const Home = () => {
+  return (
+    <MainContainer>
+      <section className="flex flex-col items-center ">
+        <h1 className=" text-4xl font-bold py-2">어서오세요🖐</h1>
+        <p>기술 면접을 위한 문제은행입니다!</p>
+      </section>
+      <section className="md:flex justify-around items-center py-4 ">
+        {[...Array(3)].map((test) => (
+          <IndexCard key={test} />
```

Comment:
```suggestion
        {[...Array(3)].map((test, index) => (
          <IndexCard key={"test" + index} />
```

이게 조금 더 조와보여요!!! 비커즈 혹시 모르게 키값이 중복될수도 있다보니가 인덱스루 처리하는게 지금은 조아요